Adds the Caravan-Class

AquillaF opened the pull request:
Adds the Caravan-Class Modular (but not really) ship.

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Adds the Caravan-Class, a ship meant for a smallish crew of 6 people.

Lore wise its meant as a modular ship. With each

 section being able to be changed out to use the user's needs. This configuration is meant as a independent general purpose ship.

![Caravan](https://user-images.githubusercontent.com/88289697/168163122-9eff779c-af0d-4347-8eaa-0c6505c369d8.png)

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
More variety in ships is always a nice thing.

## Changelog
:cl:
add: Caravan-Class Modular Ship
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

AquillaF committed:
Adds the Caravan-Class

Adds the Caravan-Class Modular (but not really) ship.

Vexylius commented on _maps/shuttles/shiptest/IndiCaravanTypeA.dmm:
```diff
@@ -0,0 +1,3448 @@
+"kb" = (
+/obj/machinery/atmospherics/components/trinary/filter/flipped/on/layer4{
+	dir = 4;
+	filter_type = "n2
```
missing "

AquillaF committed:
Fixes typo :^)

oops

Apogee-dev commented on _maps/configs/IndiCaravanTypeA.json:
```diff
@@ -0,0 +1,26 @@
+{
+	"map_name": "Caravan-class Modular ship",
+	"map_short_name": "Caravan-class",
+	"map_path": "_maps/shuttles/shiptest/IndiCaravanTypeA.dmm",
```
"prefix": "ISV",
"namelists": insert namelists of choice here, recommend GENERAL, SPACE at minimum

AquillaF replied:
I knew I was forgetting something.

Apogee-dev commented on _maps/configs/IndiCaravanTypeA.json:
```diff
@@ -0,0 +1,26 @@
+		},
+		"Scientist": 1,
+		"Medical Doctor": 1,
+		"Atmospheric Technician": 1,
```
you may want to use custom outfits and titles for these three specialist jobs, as the defaults for all three are rather NT-themed. Even if you stick to the default outfits a custom title still adds some useful flavor.

AquillaF committed:
Fixed prefix + ship names and changed job titles

prefix is now ISV
names pull from GENERAL and SPACE
Medical Doctor -> Ship's Doctor
Atmospheric Technician -> Engine Technician
Scientist -> Fauna Researcher

AquillaF committed:
removed some extra pipes from reality

had some pipes placed on top of some pipes

Apogee-dev commented:
I've just realized that the ship's pet does not have a custom name. This is a critical oversight and must be rectified at once

AquillaF committed:
Fixes some atmos pipeing

AAAAAAAAAAA

AquillaF commented:
oh yeah, you are right.I cant believe I forgot that.

AquillaF committed:
The most important update, cat name

Gives the cat a name.
also fixed me being a bit over-zealous with fixing some pipe stuff, oops.

AquillaF commented:
New cat name just dropped, Félicette! The first and only cat to have been in space!

AquillaF committed:
Lmao I forgot pickaxes

kind of hard to get glass without a pickaxe or shovel.

AquillaF committed:
Fixes air mixer pressure

air mixer was set to 4500kpa and would drain the tank was faster then intended, leaving no o2 for the TEG, now its set to 101kpa.

AquillaF committed:
added some QoL stuff

Adds a portable air pump and scrubber to engineering.
Replaced one of the bathrooms with a janitor closet

Latentish commented:
While I like this ship, I do feel like it suffers from a lot of squares and boxes. Is there a way you could possibly alter it's outer hull/silhouette to help remedy that? If not I suppose it's more of a minor issue with this ship.

Apogee-dev commented:
> While I like this ship, I do feel like it suffers from a lot of squares and boxes. Is there a way you could possibly alter it's outer hull/silhouette to help remedy that? If not I suppose it's more of a minor issue with this ship.

I don’t see any problem with the boxiness, especially considering it’s supposed to be a modular, industrial ship. The overall layout still has enough variance to maintain interest and stuff like the gaps and catwalks going on even if it’s made of boxes put together.

AquillaF commented:
I did do a small test with making it a bit less r e c t a n g u l a r for the forward module. I'm not sure how much I like it though (also testing how wall girders look rather then the lattice + railing).

![Caravan2](https://user-images.githubusercontent.com/88289697/171036919-2192a211-48be-4b18-83ac-9378de333471.png)

AquillaF committed:
expands the crew/bridge module a bit

what the title says, also changes a few things with the "linkages" for the modules.
